.solr_wrapper.yml

```diff
@@ -1,9 +0,0 @@
-# Place any default configuration for solr_wrapper here
-# port: 8983
-version: 7.4.0
-collection:
-  dir: solr/conf/
-  name: blacklight-core
-  persist: true
-instance_dir: tmp/solr/
-download_dir: tmp/solr_dl
```

Comment:
we still need this file for development right? just until we move to docker on local..